### ISSUE TITLE
0.0.6 tries to require unavailable libraries in 1.9.2

### ISSUE THREAD

**Issue by ylansegal** [sporkrb/spork-testunit#26]
After upgrading to 0.0.6, test no longer run on 1.9.2:

```
$testdrb -I test/ test/unit/address_test.rb
/Users/ylansegal/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in `require': no such file to load -- test/unit/testresult (LoadError)
    from /Users/ylansegal/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
    from /Users/ylansegal/.rvm/gems/ruby-1.9.2-p290@locum/gems/spork-testunit-0.0.6/bin/testdrb:7:in `<top (required)>'
    from /Users/ylansegal/.rvm/gems/ruby-1.9.2-p290@locum/bin/testdrb:19:in `load'
    from /Users/ylansegal/.rvm/gems/ruby-1.9.2-p290@locum/bin/testdrb:19:in `<main>'
```

As you can see, I am using ruby 1.9.2. After I downgrade to 0.0.5, everything works as expected:

```
$ testdrb -I test/ test/unit/address_test.rb
Loaded suite ["test/", "test/unit/address_test.rb"]
Started
..
Finished in 1.887994 seconds.

2 tests, 16 assertions, 0 failures, 0 errors, 0 skips

Test run options: --seed 29527
==============================================================================
```

As I side note for others that might have this problem. Even though I specifically asked for 0.0.5 in my Gemfile and ran bundle again, I was still seeing the same problem because I was neglecting to run 'bundle exec' before testdrb.

**Comment by ghost** [comment]
+1 problem confirmed. I faced same issue. Version 0.0.5 saved me.

**Comment by rmm5t** [comment]
+1 Same problem here.

**Comment by rmm5t** [comment]
Btw, if you use this fork, you'll get working Ruby 1.9.3 support too.  Related to #17

https://github.com/oriolgual/spork-testunit

**Comment by jmacdonald** [comment]
+1 also ran into the issue.

**Comment by uhlenbrock** [comment]
+1

**Comment by timcharper** [comment]
fixed in 0.0.7. Someone want to step up to maintain this gem? I merged a patch that broke it. I don't use it so it makes it difficult for me to tell which patches are going to break it.

**Comment by nixme** [comment]
@timcharper. I'd be up for maintaining the gem. We're pretty heavy users of spork + test::unit/minitest at work.
